app/models/event.py

```diff
@@ -24,6 +24,7 @@
 from app.models.ticket_fee import get_fee
 from app.models.ticket_fee import get_maximum_fee
 from app.models.ticket_holder import TicketHolder
+from app.models.order import Order, OrderTicket
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'Order' from line 18](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4801455)